amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2314,6 +2314,112 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/holds/{holdId}/bulk-statuses':
+    get:
+      tags:
+        - holds
+      operationId: listBulkStatuses
+      summary: Get bulk statuses
+      description: |
+        Gets bulk statuses for hold with id **holdId**.
+      parameters:
+        - $ref: '#/parameters/holdIdParam'
+        - $ref: '#/parameters/skipCountParam'
+        - $ref: '#/parameters/maxItemsParam'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/HoldBulkStatusPaging'
+        '400':
+          description: |
+            Invalid parameter: **holdId** is not a valid format
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission to read **holdId**
+        '404':
+          description: "**holdId** does not exist"
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  '/holds/{holdId}/bulk-statuses/{processId}':
```

Comment:
Maybe this?
```suggestion
  '/holds/{holdId}/bulk-statuses/{bulkStatusId}':
```

Reply:
I agree, `bulkStatusId` is more descriptive. I replaced all references.